README.md

```diff
@@ -21,6 +23,12 @@ ALTER USER sovisuplus CREATEDB;
 CREATE DATABASE sovisuplus;
 GRANT ALL PRIVILEGES ON DATABASE sovisuplus to sovisuplus;
 ```
+In case you are using PostgreSQL v15 the `CREATE` privilege do not grant  to everyone on the public schema by default.to resolve this problem try using the following command 
+
+```sql 
+GRANT CREATE ON SCHEMA public TO sovisuplus;
```

Comment:
This command was not enough (or even no required) to fix the issue. We had to grant  the ownership on the database to sovisuplus.